src/components/Loans/OpenLoans/components/Modals/Modals.test.js

```diff
@@ -0,0 +1,148 @@
+import okapiCurrentUser from 'fixtures/okapiCurrentUser';
+
+import renderWithRouter from 'helpers/renderWithRouter';
+import Modals from './Modals';
+
+import '__mock__/stripesCore.mock';
+import '__mock__/stripesSmartComponent.mock';
+
+jest.unmock('@folio/stripes/components');
+
+const renderModals = (props) => renderWithRouter(<Modals {...props} />);
+const STRIPES = {
+  connect: (Component) => Component,
+  config: {},
+  currency: 'USD',
+  hasInterface: () => true,
+  hasPerm: jest.fn().mockReturnValue(true),
+  clone: jest.fn(),
+  setToken: jest.fn(),
+  locale: 'en-US',
+  logger: {
+    log: () => {},
+  },
+  okapi: {
+    tenant: 'diku',
+    url: 'https://folio-testing-okapi.dev.folio.org',
+    translations: {
+      'stripes-components.Datepicker.calendar': 'calendar',
+      'stripes-components.Datepicker.calendarDaysList': 'calendar days list.',
+      'stripes-core.button.cancel': [{ type: 0, value: 'Cancel' }],
```

Comment:
Given you import all the stripes-core and stripes-components translations in `renderWithRouter` below, why is it necessary to include the translations array here too?